firefox-ios/Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -368,56 +368,39 @@ class HistoryPanel: UIViewController,
 
             if var historyActionable = item as? HistoryActionablesModel {
                 historyActionable.configureImage(for: windowUUID)
-                guard let cell = tableView.dequeueReusableCell(
-                    withIdentifier: OneLineTableViewCell.cellIdentifier,
-                    for: indexPath
-                ) as? OneLineTableViewCell
-                else {
-                    self.logger.log("History Panel - cannot create OneLineTableViewCell for historyActionable",
-                                    level: .debug,
-                                    category: .library)
-                    return nil
-                }
-
-                let actionableCell = self.configureHistoryActionableCell(historyActionable, cell)
-                return actionableCell
+                return self.getHistoryActionableCell(historyActionable: historyActionable, indexPath: indexPath)
```

Comment:
you can remove `self.` references in this function to clean up the code a little!

Reply:
Sure! Also, I'm going to remove `self.` from `getSiteCell` and `getGroupCell` too.